src/components/MessagesList/MessagesGroup/Message/MessagePart/Forwarder.vue

```diff
@@ -99,16 +100,37 @@ export default {
 		selectedConversationName() {
 			return this.$store.getters?.conversation(this.selectedConversationToken).name
 		},
+
+		/**
+		 * Object containing all the mentions in the message that will be forwarded
+		 * @returns {Object} mentions.
+		 */
+		mentions() {
+			const mentions = {}
+			for (const key in this.messageObject.messageParameters) {
+				if (key.startsWith('mention')) {
```

Comment:
please use the param type to see if it's a "user", "guest" or "call" mention

there's some example code here: https://github.com/nextcloud/spreed/blob/bugfix/noid/forward-message-mention/src/store/messagesStore.js#L59

Reply:
I noticed that if I replace with .name instead of with .id, both room and user mentions work. I just pushed the change

Reply:
Only if displayname === id, you have to use id.

Reply:
I don't understand this either, If in that scenario `displayName === id`, why can't I just use displayName always as I'm doing here?